### DISCUSSION TITLE
Some streams start 6 hours in the past

### DISCUSSION THREAD

**Discussion by benevenstanciano** [iptv-org/community#205] [Q&A]
Hi everyone,
Long time user/tinkerer, first time commenter.
I'm a big fan of everybody's work and discussions, so thank you first of all.
For some streams, specifically those in the format theus6tv.tk/locals/[place]/[channel].m3u8, when I open in VLC, they are live. Yet when I move the streams to my linux-based TV receiver, the stream starts 6 hours in the past. Is there some kind of autotimeshift in place? I'm not sure if I can get it to go live by messing around with the URL.
Thanks!

**Comment by jkemp814** [comment]
Check your EPG settings in your Linux receiver. Kodi will do the same thing in the iptv-simple-client. Set it to '0' hours, if yours has that setting.

**Reply by MapGuy11** [reply]
The source I use is a buffered OTA Source so the stream will start 6hours previous (Catch Up type thing) or when the last blip in OTA signal. Though it should bring you right to live.

**Reply by benevenstanciano** [reply]
It's very strange. It must be an issue with the player on the linux receiver, because when I access the receiver remotely through another device and launch the stream it's live. 
Weirdly, WABC isn't affected. All other channels seem to be.
